.pre-commit-config.yaml

```diff
@@ -9,13 +9,13 @@ repos:
   - id: check-added-large-files
 - repo: local
   hooks:
-    - id: cargo-fmt-nightly
-      name: Cargo Fmt Nightly
+    - id: cargo-fmt
```

Comment:
drive-by to fix the CI failure. luckily this doesn't cause any formatting changes in other files.